SocialCareCaseViewerApi.Tests/V1/Helpers/TestHelpers.cs

```diff
@@ -96,11 +96,58 @@ public static CaseNoteResponse CreateCaseNoteResponse(CaseNote? caseNote)
                 .RuleFor(c => c.AllocatedTeamId, f => teamId ?? team.Id)
                 .RuleFor(c => c.AllocatedWorkerId, f => workerId ?? worker.Id)
                 .RuleFor(c => c.CreatedBy, f => createdBy ?? createdByWorker.Email)
+                .RuleFor(c => c.RagRating, f => "urgent")
```

Comment:
simple way to add randomness against known values is to setup a list with string values and then use `f.PickRandom(stringValuesList)` to pick one at random